tests/virtualization/yast_virtualization.pm

```diff
@@ -28,12 +35,21 @@ sub run() {
     if (get_var("STANDALONEVT")) {
         assert_screen "virt-sle12sp1-gnome_yast_virtualization_OK", 200;
     }
-    else {
+    elsif (check_var("FLAVOR", "Server-DVD")) {
         assert_screen "virt-sle12sp1-gnome_yast_virtualization_bridge", 200;
         # select yes
         send_key "alt-y";
     }
-    send_key "alt-o";
+
+    if (check_var("FLAVOR", "Desktop-DVD")) {
```

Comment:
Is this needle really so SLES/SLED specific that it need unique tag ?
Without `if FAVOR ...` it would be just two lines of code and needle can be separated with ENV-* tag